vendor/README.md

```diff
@@ -595,6 +595,7 @@ This is a list of grammars that Linguist selects to provide syntax highlighting
 - **TypeScript:** [tree-sitter/tree-sitter-typescript](https://github.com/tree-sitter/tree-sitter-typescript) 🐌
 - **TypeSpec:** [microsoft/typespec](https://github.com/microsoft/typespec)
 - **Typst:** [michidk/typst-grammar](https://github.com/michidk/typst-grammar)
+- **URScript:** [MagicStack/MagicPython](https://github.com/MagicStack/MagicPython)
```

Comment:
Huh!? This file is auto-updated so this shouldn't be pointing to the MagicPython grammar.

Remove the line and run `script/list-grammars` - this should fix things.